data/lists/topiclist.xyz.json

```
@@ -0,0 +1,34 @@
+{
+    "id": "topiclist.xyz",
+    "name": "TopicList",
+    "url": "https://topiclist.xyz/",
+    "icon": "https://cdn.topiclist.xyz/images/png/TopicList5.png",
+    "language": "English",
+    "display": 1,
+    "defunct": 0,
+    "discord_only": 0,
+    "description": "Do you want to expand and improve your Discord bot? TopicList is here for you!",
+    "api_docs": "https://docs.topiclist.xyz",
+    "api_post": "https://api.topiclist.xyz/bot/:id/stats",
+    "api_field": "guilds",
+    "api_shard_id": null,
+    "api_shard_count": "shards",
+    "api_shards": null,
+    "api_get": "https://topiclist.xyz/api/bots/:botID",

```

```suggestion
    "api_get": "https://topiclist.xyz/api/bots/:id",
```
The placeholder for the bot ID should always be `:id`